rest/servers/spring-boot/pom.xml

```diff
@@ -71,10 +49,19 @@
         <dependency>
             <groupId>io.springfox</groupId>
             <artifactId>springfox-swagger2</artifactId>
+            <version>${springfox.version}</version>
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
+            <version>${spring.boot.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
```

Comment:
❗️ I would caution against doing this. The reason I didn't originally is because you loose the control of selecting versions since you inherit them through this dependency and that did cause problems in some cases with open-api as it could with other dependencies you require.

Reply:
This was added by accident. Good catch.

Reply:
How many dependences caused problems? If it is only a few we can use <exclude> to exclude them from the Spring Boot artifact, then use <dependency> to include the right version.